resources-domain/resources-service/package.json

```diff
@@ -46,6 +46,7 @@
     "es:delete-index:local": "cross-env AWS_REGION=us-east-1 SSM_ENDPOINT=http://localhost:4566 NODE_ENV=local ELASTICSEARCH_CONFIG_node=http://localhost:9200 tsx ./scripts/elasticsearch/delete-index.ts",
     "build-scripts": "tsc --project tsconfig.scripts.json",
     "reindex": "tsx ./scripts/elasticsearch/reindexResources.ts",
+    "reindex-compiled": "node ./dist/scripts/elasticsearch/reindexResources.js",
```

Comment:
when do we need to run this "new" script?

In the README file we're running like this:
```
node ./node_modules/@tech-matters/resources-service/scripts-dist/scripts/elasticsearch/reindexResources.js [...args]
```

Should we change it to use `npm reindex-compiled`?

Reply:
It's redundant, removed